packages/performance/src/services/perf_logger.test.ts

```diff
@@ -301,5 +302,78 @@ describe('Performance Monitoring > perf_logger', () => {
         EXPECTED_NETWORK_MESSAGE
       );
     });
+
+    it('skips performance collection if domain is cc', () => {
```

Comment:
Fireperf automatically collects network request (including request to cc). However, to avoid infinite loop of network request collection (request to CC -> request is captured as performance resource -> another request to CC), if the domain is cc, then we will not collect the network request.

Reply:
Thanks James. Based on our offline discussion this is validating that we don't instrument Network Request to CC/FL sent by our own SDK.

Since it took a while to understand this scenario it would be good to add some documentation to make it clear.

Reply:
Thank you Raman! Added comments to explain that we don't instrument requests sent from SDK itself in both production code and test code.